server/image_status.go

```diff
@@ -34,7 +36,15 @@ func (s *Server) ImageStatus(ctx context.Context, req *types.ImageStatusRequest)
 	}
 
 	if status == nil {
-		log.Infof(ctx, "Image %s not found", img.Image)
+		if artifact, err := s.ArtifactStore().Status(ctx, img.Image); err == nil {
+			return &types.ImageStatusResponse{
+				Image: artifact.CRIImage(),
+			}, nil
+		} else if !errors.Is(err, ociartifact.ErrNotFound) {
+			log.Errorf(ctx, "Unable to get artifact: %v", err)
+		}
+
+		log.Infof(ctx, "Neither image nor artfiact %s found", img.Image)
```

Comment:
It doesn't need else because it returns in the previous if block.
It logs twice when the error is not ErrNotFound.

```suggestion
		}
		if errors.Is(err, ociartifact.ErrNotFound) {
			log.Infof(ctx, "Neither image nor artfiact %s found", img.Image)
		} else {
			log.Errorf(ctx, "Unable to get artifact: %v", err)
		}
```